backend/tournesol/serializers/rating.py

```diff
@@ -51,24 +51,24 @@ def to_internal_value(self, data):
 
 
 class ContributorRatingCreateSerializer(ContributorRatingSerializer):
-    video_id = RegexField(YOUTUBE_VIDEO_ID_REGEX, write_only=True)
+    uid = RegexField(rf"yt:{YOUTUBE_VIDEO_ID_REGEX[1:]}", write_only=True)
```

Comment:
Tweaking the regex here does not seem very robust, and very specific to videos. Maybe we could remove this regex validation, and rely on the `RelatedEntitySerializer` to do the necessary checks (depending on the current poll, etc.)

Reply:
Good, I've used a `CharField` instead.

Will rework how `uid` validation works in the next PR.